pylint/reporters/json_reporter.py

```diff
@@ -196,6 +200,120 @@ def serialize_stats(self) -> dict[str, str | int | dict[str, int]]:
         }
 
 
+class SARIFReporter(BaseReporter):
```

Comment:
Should be in ``pylint/reporters/sarif_reporter.py`` not ``pylint/reporters/json_reporter.py``